authors/detail.php

```diff
@@ -0,0 +1,60 @@
+<?php
+include '../csv_util.php';
+$line_counter = 0;
+$index = $_GET['index'];
+$authors_array = read_csv('../data/authors.csv');
+$author_name = $authors_array[$index];
+
+function printQuote($filename,$index){
```

Comment:
`$filename` is not used in the function

Reply:
should be fix with the most recent commit